<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="clientsGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Client definitions</string>
    </property>
    <widget class="QScrollArea" name="clientsScrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>601</width>
       <height>401</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>599</width>
        <height>399</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="clientFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>601</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of client types to add:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="clientCountSpinBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>59</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Define number of client types to add.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeClientButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Remove client types from list.</string>
      </property>
      <property name="text">
       <string>Remove clients</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addClientButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add defined amount of new client types. </string>
      </property>
      <property name="text">
       <string>Add new client(s)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="streamsGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Stream definitions</string>
    </property>
    <widget class="QScrollArea" name="streamsScrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>611</width>
       <height>401</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>609</width>
        <height>399</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="streamFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>611</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="removeStreamButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>191</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove stream</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addStreamButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>191</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Add stream</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Application layer protocol</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>311</width>
       <height>191</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QSpinBox" name="appProto_ackSize">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>The acknowledgement payload size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="appProto_headerSize">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Size of the application protocol header.</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="appProto_RTO">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>The time limit when the message is retransmitted.</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="appProto_delAck">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Waiting time before an acknowledgement is sent.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Acknowledgement size (bytes):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Header size (bytes):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Retransmission timeout (ms):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Delayed ack value (ms):</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>311</width>
        <height>61</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The application protocol must be enabled separately for streams that use it, otherwise these configurations have no effect.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Simulation configuration</string>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>311</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Simulation time (seconds):</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="simTime">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Defines the simulation execution time.</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>streamsGroupBox</zorder>
   <zorder>clientsGroupBox</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
